dnSpy/dnSpy/Themes/wpf.styles.templates.xaml

```diff
@@ -2085,6 +2085,56 @@
             </Trigger>
         </Style.Triggers>
     </Style>
+
+    <Style x:Key="SortableColumnHeaderStyle"
+           BasedOn="{StaticResource {x:Type GridViewColumnHeader}}"
+           TargetType="{x:Type GridViewColumnHeader}">
+        <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+    </Style>
+
+    <DataTemplate x:Key="SortableColumnHeaderTemplate">
+        <DataTemplate.Resources>
+            <Style TargetType="Path">
+                <Setter Property="Margin" Value="10,0,0,0" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="StrokeThickness" Value="1" />
+                <Setter Property="Fill" Value="{DynamicResource HeaderDefaultText}" />
+            </Style>
+        </DataTemplate.Resources>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+            <TextBlock Grid.Column="0" VerticalAlignment="Center" TextAlignment="Left" Text="{Binding}" />
+            <Path Grid.Column="1" Name="DownArrow" Data="M 0,0 10,0 5,5 Z" />
```

Comment:
I think you can do this with just one `Path` if your triggers below just modify the `Data` property